firefox-ios/Client/Ecosia/UI/NTP/MultiPurposeHeader/NTPMultiPurposeEcosiaHeaderViewModel.swift

```diff
@@ -17,10 +17,7 @@ final class NTPMultiPurposeEcosiaHeaderViewModel: ObservableObject {
     private let windowUUID: WindowUUID
     internal weak var delegate: NTPMultiPurposeEcosiaHeaderDelegate?
     internal var theme: Theme
-
-    // MARK: - Theme Properties (following FeedbackView pattern)
-    @Published var buttonBackgroundColor = Color.gray.opacity(0.2)
-    @Published var buttonIconColor = Color.primary
+    @SwiftUICore.Environment(\.themeManager) var themeManager
```

Comment:
I guess this is not needed here since handled within the ThemeModifier now via DI, right?

Reply:
True! Indeed part of the old one, removing it!